example/processors/pii/pii.py

```diff
@@ -41,16 +42,19 @@ def return_report(csv):
     for key, details in analysis.items():
         if details:
             code = 'check_pii_detail:pii-found:{key}'.format(key=key)
-            tabular_add_issue(
+            report.TabularReport.add_issue(
```

Comment:
As in csv_checker, make sure you are using an actual object, rather than calling a function statically inside the class -- you are passing `self._report` to `return_report` in line 72, but up in line 31, you still have `return_report(csv)`, rather than `return_report(csv, rprt)`, for example. If you fix that up, then `rprt` is your object, and you can call `rprt.add_issue(...)` anywhere in this function.

I appreciate the `get_workflow` syntax makes this even more confusing, but feel free to hit me up for answers if you've any trouble getting it running!